envs/monkey_zoo/blackbox/test_blackbox.py

```diff
@@ -1,11 +1,13 @@
 import logging
 import os
+from copy import deepcopy
 from http import HTTPStatus
 from threading import Thread
 from time import sleep
 from uuid import uuid4
 
 import pytest
+import requests
```

Comment:
I don't see where this is being used

Reply:
Down below. `with pytest.raises(requests.exceptions.HTTPError)`